aspnetcore/fundamentals/minimal-apis/handle-errrors.md

```diff
@@ -0,0 +1,165 @@
+
+``` csharp
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.Map("/exception", () 
+    => { throw new InvalidOperationException("Sample Exception"); });
+
+app.Run();
+```
+
+### Developer Exception Page
+
+The [Developer Exception Page](xref:fundamentals/error-handling#developer-exception-page) shows detailed stack traces for server errors. It uses <xref:Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware> to capture synchronous and asynchronous exceptions from the HTTP pipeline and to generate error responses. 
+
+ASP.NET Core apps enable the developer exception page by default when both:
+
+* Running in the [Development environment](xref:fundamentals/environments).
+* App created with the current templates, that is, using [WebApplication.CreateBuilder](/dotnet/api/microsoft.aspnetcore.builder.webapplication.createbuilder).
+
+For more information on configuring middleware see, [Middleware in Minimal API apps](/aspnet/core/fundamentals/minimal-apis/middleware).
+
+Using the preceding Minimal API app, when the `Developer Exception Page` detects an unhandled exception, it generates a default plain-text response similar to the following example:
+
+```console
+HTTP/1.1 500 Internal Server Error
+Content-Type: text/plain; charset=utf-8
+Date: Thu, 27 Oct 2022 18:00:59 GMT
+Server: Kestrel
+Transfer-Encoding: chunked
+ 
+    System.InvalidOperationException: Sample Exception
+    at Program.<>c.<<Main>$>b__0_1() in ....:line 17
+    at lambda_method2(Closure, Object, HttpContext)
+    at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
+    --- End of stack trace from previous location ---
+    at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
+HEADERS
+=======
+Accept: */*
+Connection: keep-alive
+Host: localhost:5239
+Accept-Encoding: gzip, deflate, br
+```
+
+> [!WARNING]
+> Don't enable the Developer Exception Page **unless the app is running in the Development environment**. Don't share detailed exception information publicly when the app runs in production. For more information on configuring environments, see <xref:fundamentals/environments>.
+
+### Exception handler
+
+In non-development environments, use [Exception Handler Middleware](xref:fundamentals/error-handling#exception-handler-page) to produce an error payload. To configure the `Exception Handler Middleware` in the preceding Minimal API app, call <xref:Microsoft.AspNetCore.Builder.ExceptionHandlerExtensions.UseExceptionHandler%2A>. 
+
+For example, the following code change the app to respond an [RFC 7807](https://tools.ietf.org/html/rfc7807)-compliant payload to the client. For more information, see [Problem Details](#problem-details) section.
+
+
+``` csharp
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.UseExceptionHandler(exceptionHandlerApp 
```

Comment:
@tdykstra @Rick-Anderson All sample codes here are very simple, so, I decided to add them directly. Do you know if there is any possibility to highlight lines in this case? If not, maybe I will move them to a small project to have it.

Reply:
> Do you know if there is any possibility to highlight lines in this case? If not, maybe I will move them to a small project to have it.

Not at the moment. If you want put `<!-- comments in the MD file where it should be highlighted -->` we can create the sample and do the migration for you.